packages/default/scss/timeline/_layout.scss

```diff
@@ -0,0 +1,388 @@
+.k-timeline {
+    width: 100%;
+    padding: 0 $timeline-padding-x;
+    margin: $timeline-padding-y 0;
```

Comment:
Rename this variable.
...
A possible solution is to rename both variables to something like:
$timeline-spacing-y
$timeline-spacing-x